src/test/java/StringTest.java

```diff
@@ -0,0 +1,35 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("split")
+    void requirementFirst() {
+        String firstStr = "1,2";
+        String secondStr = "1";
+
+        assertThat(firstStr.split(",")).isEqualTo(new String[]{"1", "2"});
```

Comment:
```suggestion
        assertThat("1,2".split(",")).isEqualTo(new String[]{"1", "2"});
```

이렇게 간단한 테스트는 굳이 로컬 변수로 구현하지 않아도 되지 않을까?

Reply:
확인 감사드립니다!~